yarn-project/circuits.js/src/structs/private_circuit_public_inputs.ts

```diff
@@ -237,11 +267,19 @@ export class PrivateCircuitPublicInputs {
       fields.version,
     ] as const;
   }
+
   /**
    * Serialize this as a buffer.
    * @returns The buffer.
    */
   toBuffer(): Buffer {
     return serializeToBuffer(...PrivateCircuitPublicInputs.getFields(this));
   }
+
+  /**
+   * Serialize this as a field array.
+   */
+  toFields(): Fr[] {
+    return serializeToFieldArray(...PrivateCircuitPublicInputs.getFields(this));
```

Comment:
For the functions that have `getFields` should we generally be using this approach instead? For some cases higher up, there was both a `getFields` and a `toFields` which both were setting the order, might be better to do it like here to not run into ordering issues.

Reply:
Agree, will update CallContext here and follow that approach from on.